apps/web/src/pages/ApplicantDashboardPage/components/QualifiedRecruitments/QualifiedRecruitments.tsx

```diff
@@ -0,0 +1,216 @@
+            <p>
+              {intl.formatMessage({
+                defaultMessage:
+                  "<strong>Expired</strong> recruitments are processes that have ended either because they’ve become stale or because a new, more up-to-date recruitment has replaced them. You will not show up in results related to expired recruitments.",
+                id: "qYjmRp",
+                description:
+                  "Expired list item for my qualified recruitments section on the applicant dashboard.",
+              })}
+            </p>
+          </li>
+        </ul>
+      </div>
+      <div>
+        <Accordion.Root
+          type="multiple"
+          mode="simple"
+          value={currentAccordionItems}
+          onValueChange={(newValue: AccordionItems) => {
+            setCurrentAccordionItems(newValue);
+          }}
+        >
+          {/* Active Recruitments */}
+          <Accordion.Item value="active">
+            <Accordion.Trigger>
+              {intl.formatMessage({
+                defaultMessage: "Active recruitments",
+                id: "lfZeyc",
+                description:
+                  "Heading for active recruitments accordion on the applicant dashboard.",
+              })}
+            </Accordion.Trigger>
+            <AnimatedContent isOpen={currentAccordionItems.includes("active")}>
+              {activeRecruitments.length > 0 ? (
+                activeRecruitments.map((activeRecruitment) => (
+                  <QualifiedRecruitmentCard
+                    key={activeRecruitment.id}
+                    application={activeRecruitment}
+                  />
+                ))
+              ) : (
+                <Well data-h2-text-align="base(center)">
+                  <p
+                    data-h2-font-size="base(h5)"
+                    data-h2-font-weight="base(700)"
+                    data-h2-margin="base(0, 0, x.25, 0)"
+                  >
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "Recruitments you've been accepted into will appear here.",
+                      id: "VUjh3i",
+                      description:
+                        "Text displayed in active qualified recruitments section when empty.",
+                    })}
+                  </p>
+                  <p data-h2-font-size="base(h6)">
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "Recruitments in this section are actively being hired from.",
+                      id: "eGPSG3",
+                      description:
+                        "Additional text displayed in active qualified recruitments section when empty.",
+                    })}
+                  </p>
+                </Well>
+              )}
+            </AnimatedContent>
+            {/* Expired Recruitments */}
+          </Accordion.Item>
+          <Accordion.Item value="expired">
+            <Accordion.Trigger>
```

Comment:
Same thing here for `headerAs="h3"`

Reply:
Fixed by [997d65f](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5988/commits/997d65f03087c750485b174c0aeaa08aeec6b1e0)